pycbc/psd/variation.py

```diff
@@ -131,7 +131,7 @@ def calc_filt_psd_variation(strain, segment, short_segment, psd_long_segment,
     # Create a bandpass filter between low_freq and high_freq
     filt = sig.firwin(4 * srate, [low_freq, high_freq], pass_zero=False,
                       window='hann', nyq=srate / 2)
-    filt.resize(int(psd_duration * srate))
+    filt.resize(int(psd_duration * srate), refcheck=False)
```

Comment:
Why would this need `refcheck=False`? The array is only created in the line above, so it shouldn't be being referenced from anywhere else.

Reply:
I don't remember exactly why I made this change but it was most likely due to it not working without the change. This could be very old. I will remove the refcheck in both the offline and live versions and see if there are any errors.

Reply:
No errors from what I can see so I have removed it.